OJP/OJP_Trips.xsd

```diff
@@ -1304,7 +1339,7 @@
 		</xs:annotation>
 		<xs:sequence>
 			<xs:group ref="BaseTripPolicyGroup"/>
-			<xs:element name="MultiPointType" type="MultiPointTypeEnumeration" default="anyPoint" minOccurs="0">
```

Comment:
Why does this one change?

Reply:
I think not having a MultiPointType is generally bad. It is the explaining difference.  Do you think differently?

Reply:
To be honest: I have no clue what it does ;-)

Reply:
I defines the behaviour of the MPTR regarding its origins and destinations